pkg/render/manager.go

```diff
@@ -195,6 +204,9 @@ func (c *managerComponent) managerDeployment() *appsv1.Deployment {
 	}
 	if c.management {
 		annotations[voltronTunnelHashAnnotation] = AnnotationHash(c.tunnelSecrets[0].Data)
+		if len(c.tunnelSecrets) == 2 {
```

Comment:
This seems kindof flimsy. Should we really be grouping the internalSecret in with `c.tunnelSecrets`? Also wondering why `c.tunnelSecrets` is a slice in the first place. Should we just have `c.tunnelSecret` and `c.internalSecret`?

Reply:
I think that would be better. I do not really like the fact that all of these are slices and we only populate the first element and use it to generate hashes.

Reply:
Idk I see you're mostly just following the pattern here. Seems odd that we love setting up slices of secrets when we usually expect the length to just be 1 or 0. I'm open to keeping it as is...